src/svgstore.js

```diff
@@ -53,11 +53,28 @@ function svgstore(options) {
 					var oldDefId = elem.attr('id');
 					var newDefId = id + '_' + oldDefId;
 					elem.attr('id', newDefId);
+
+					/* process use tags */
 					child('use').each(function (i, use) {
-						if (child(use).prop('xlink:href') !== '#' + oldDefId) {
+						var hrefLink = '#' + oldDefId;
+						var checkableProperties = ['xlink:href', 'href'];
+						var foundProperty;
+						for (var j = 0; j < checkableProperties.length; j++) {
+							var currentProperty = checkableProperties[j];
+							if (child(use).prop(currentProperty) === hrefLink) {
+								foundProperty = currentProperty;
+								break;
+							}
+						}
+						if (!foundProperty) {
 							return;
 						}
-						child(use).attr('xlink:href', '#' + newDefId);
+						child(use).attr(foundProperty, '#' + newDefId);
+					});
+
+					/* process fill attributes */
+					child('[fill="url(#' + oldDefId + ')"]').each(function (i, use) {
```

Comment:
It looks like there's a big long list of attributes that accept `url(#...)` ([FuncIRI](https://www.w3.org/TR/SVG/types.html#DataTypeFuncIRI)) as a value:

https://www.w3.org/TR/SVG/linking.html#processingIRI

Reply:
d'oh! ok, will fix this tomorrow :)

Reply:
Unfortunately, we decided not to spend the company's time on covering all the cases for now, so I'd say that I'll take care of this only when I have my own time. For now, the fix solves our problems and still helps a lot with arranging the whole icons system.
I'll try to come back to the fixes this or next weekend. Just keeping you posted :)